web/src/app/users/UserContactMethodCreateDialog.js

```diff
@@ -24,6 +40,45 @@ export default function UserContactMethodCreateDialog(props) {
     value: '',
   })
 
+  const [submitted, setSubmitted] = useState(false)
+
+  const { data, loading: queryLoading } = useQuery(query, {
+    variables: {
+      input: {
+        cmValue: cmValue.value,
+        cmType: cmValue.type,
+      },
+    },
+  })
+
+  const [userConflict, setUserConflict] = useURLParam('userConflict', '')
+
+  const resetUserConflict = useResetURLParams('userConflict')
```

Comment:
Don't use query params for this, use the error as the source of the query, then we don't need to track redundant data through state & params.